src/lib/util/uint128.h

```diff
@@ -62,7 +62,7 @@
 /** Create a 128 bit integer value with n bits high
  *
  */
-static uint128_t uint128_gen_mask(uint8_t bits)
+UNUSED static uint128_t uint128_gen_mask(uint8_t bits)
```

Comment:
it's better to just `ifdef` them out... I thought it already did that?

Reply:
the compiler definitely will not warn about unused functions defined in headers.

Reply:
maybe change them to static inline if you're seeing issues

Reply:
`inline` did the trick, I thought that was a C++ thing that didn't exist in C